design-documents/graph-ql/coverage/ProductPrices.graphqls

```diff
@@ -0,0 +1,52 @@
+interface ProductInterface{
+    ...
+    price: ProductPrices @deprecated (use price_range)
+    price_range: PriceRange!
+    tier_price: Float @deprecated (use tier_prices)
+    tier_prices: [ProductTierPrices]
+    ...
+}
+
+type ProductPrices { @deprecated (use PriceRange)
+    minimalPrice: @deprecated
+    maximalPrice: @deprecated
+    regularPrice: @deprecated
+}
+
+type ProductTierPrices {
+    customer_group_id: String
```

Comment:
Looks like we don't need to share `customer_group_id` on frontend
Price could be resolved on backend based on headers

Reply:
website_id and customer_group_id are already present in the ProductTierPrices type. We could deprecate them if that is desired.

Reply:
group ID should be removed, GraphQL never return another group than a group of a user who invokes a query

Reply:
as well as website